LandSandBoat/server
# Convert 'Greater Colibri' and 'Wivre's to their 'Locus' successors

**_I affirm:_**

## What does this pull request do?

Begin conversion of Wivres and Greater Colibris in Bhaflau Thickets to their 'Locus' successors. 
Captures attached below:
[2022-07-15_11-53.zip](https://github.com/LandSandBoat/server/files/9577400/2022-07-15_11-53.zip)

## Steps to test these changes

Visit Bhaflau Thickets and notice that Wivres are no longer called Colibri and vice versa. 

**NOTE: I have two outstanding issues with this work.**

~~1. I cannot get the Locus Wivres to use their skill lists. The list ID's and the skills contained in them haven't changed. Also I can force the mobs to use these abilities using !exec target:useMobAbility(x), stumped on this.~~
~~2. I'm getting an error on startup that doesn't make sense to me. I'm not sure where mob ID 16990487 is being referenced. It's not anywhere in my captures, nor is it in any of our SQLs.~~
Placeholder was added to DB for 16990487 to prevent startup errors.

![image](https://user-images.githubusercontent.com/65316697/190465912-e8d41ceb-ee09-47ce-a33f-f84cf33b00e8.png)
